tests/regression/44-trier_analyzer/20-Pchain.c

```diff
@@ -2,10 +2,10 @@
 
 extern void *malloc(size_t);
 extern int putchar(int);
-extern int printf();
```

Comment:
What's the reason for removing this declaration here, but not the others or elsewhere?

I'm now thinking maybe it'd be a good idea to replace all these extern standard library definitions with actual includes to avoid any possible conflicts with types or other standard library internal trickery.